packages/venia-ui/lib/components/CartPage/ProductListing/EditModal/editModal.js

```diff
@@ -11,16 +11,20 @@ import ProductForm from './productForm';
 
 const EditModal = props => {
     const { item, setIsCartUpdating } = props;
-    const talonProps = useEditModal();
-    const { handleClose, isOpen } = talonProps;
+    const talonProps = useEditModal({ item });
+    const { handleClose, isOpen, setVariantPrice, variantPrice } = talonProps;
 
     const classes = mergeClasses(defaultClasses, props.classes);
     const rootClass = isOpen ? classes.root_open : classes.root;
 
     const bodyComponent = item ? (
         <div className={classes.body} key={item.id}>
-            <ProductDetail item={item} />
-            <ProductForm item={item} setIsCartUpdating={setIsCartUpdating} />
+            <ProductDetail item={item} variantPrice={variantPrice} />
```

Comment:
`variantPrice` is undefined if you open the edit modal and don't click on anything.

Reply:
Yes. We render `cartItem` price until a new variant is selected.

Reply:
So if the item in the cart is a variant of the default, `cartItem.price` will be the selected variant price not the default price?